projects/ngrx.io/content/guide/entity/adapter.md

```diff
@@ -105,17 +105,17 @@ import { Update } from '@ngrx/entity';
 
 import { User } from '../models/user.model';
 
-export const loadUsers = createAction('[User/API] Load Users', props&lt;{ users: User[] }&gt;());
-export const addUser = createAction('[User/API] Add User', props&lt;{ user: User }&gt;());
-export const upsertUser = createAction('[User/API] Upsert User', props&lt;{ user: User }&gt;());
-export const addUsers = createAction('[User/API] Add Users', props&lt;{ user: User }&gt;());
-export const upsertUsers = createAction('[User/API] Upsert Users', props&lt;{ users: User[] }&gt;());
-export const updateUser = createAction('[User/API] Update User', props&lt;{ user: Update&lt;User&gt; }&gt;());
-export const updateUsers = createAction('[User/API] Update Users', props&lt;{ users: Update&lt;User&gt;[] }&gt;());
-export const mapUsers = createAction('[User/API] Map Users', props&lt;{ entityMap: EntityMap&lt;User&gt; }&gt;());
-export const deleteUser = createAction('[User/API] Delete User', props&lt;{ id: string }&gt;());
-export const deleteUsers = createAction('[User/API] Delete Users', props&lt;{ id: string[] }&gt;());
-export const deleteUsersByPredicate = createAction('[User/API] Delete Users By Predicate', props&lt;{ predicate: Predicate&lt;User&gt; }&gt;());
+export const loadUsers = createAction('[User/API] Load Users', props<{ users: User[] }>());
+export const addUser = createAction('[User/API] Add User', props<{ user: User }>());
+export const upsertUser = createAction('[User/API] Upsert User', props<{ user: User }>());
+export const addUsers = createAction('[User/API] Add Users', props<{ users: User }>());
+export const upsertUsers = createAction('[User/API] Upsert Users', props<{ users: User[] }>());
+export const updateUser = createAction('[User/API] Update User', props<{ user: Update&lt;User&gt; }>());
+export const updateUsers = createAction('[User/API] Update Users', props<{ users: Update&lt;User&gt;[] }>());
```

Comment:
Hi @timdeschryver ,
A question her, this code looks incorrect. I think it should be 
export const updateUsers = createAction('[User/API] Update Users', props<{ users: Update&lt;User[]&gt; }>());

Reply:
It's fine how it is currently. The signatureis  `Update<T>[]`